src/main/webapp/content/scss/themes/_dark-variables.scss

```diff
@@ -151,6 +151,11 @@ $tooltip-color: $white;
 
 // === END BOOTSTRAP ===
 
+// Navigation Tab Bar
+$tab-bar-color: $gray-500;
+$tab-bar-color-hover: $gray-300;
+$tab-bar-color-active: #b1d0ea;
```

Comment:
Can you use some built-in color?

Reply:
I actually used an online tool for making $primary brighter and used that color. However, I just saw that there is a lighten() function that I could have used instead ^^"

Anyway, it's fixed now :)